api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -264,4 +295,498 @@ public void saveConcept_shouldForceSetFlagIfSetMembersExist() throws Exception {
 		//Then
 		assertTrue(concept.getSet());
 	}
+	
+	/**
+	 * @see ConceptServiceImpl#purgeConcept(Concept)
+	 * @Verifies should purge the concept if not being used by an obs
```

Comment:
please make all those "V" of "Verifies" small, the plugin writes small ones. sorry for this annoying thing.